### PR TITLE
fixed typos in data structures description

### PR DESCRIPTION
**Pull request by aedler-ft**

### CODE REVIEW

File: doc/appendix-de-kassensichv/data-structures/data-structures.md [hunk 1]
```diff
@@ -1,19 +1,19 @@
 ## Data Structures
 
-This chapter expands on the descriptions of the data structures covered in the Chapter ["Data Structures"](../../general/data-structures/data-structures.md) of the General Part, with country specific information applicable to the German market.
+This chapter expands on the descriptions of the country-specific covered in the Chapter ["Data Structures"](../../general/data-structures/data-structures.md) of the General Part, with country specific information applicable to the German market.
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/data-structures/data-structures.md:3]: of the country-specific Data Structures, which are covered in the respective chapter, ["Data Structures"](../../general/data-structures/data-structures.md), of the General...

File: doc/appendix-de-kassensichv/data-structures/data-structures.md [hunk 2]
```diff
@@ -1,19 +1,19 @@
...
 
 ### Receipt Request
 
 #### Single fields
 
-Following chapter highlights fields from the receipt request that need a special handling for the german market.
+Following chapter highlights fields from the receipt request that need special handling for the German market.
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/data-structures/data-structures.md:9]: unless I misunderstand the context here, IMO it should be
The chapter highlights fields from the receipt request that need special handling for the German market are listed below:

File: doc/appendix-de-kassensichv/data-structures/data-structures.md [hunk 3]
```diff
@@ -23,55 +23,55 @@ If you need to provide customer data in your request, you can send it in via the
...
+| `CustomerVATId` | `string(15)` | Optional | **VAT-ID of the beneficiary customer.**<br />Send via `cbCustomer` in JSON format by adding the key value pair `CustomerVATId` e.g. `"cbCustomer":"{..., "CustomerVATId":"DE123456789", ...}"` | 1.3 |
 
 
 #### Receipt case data `ftReceiptCaseData`
 
-In the general description the field `ftReceiptCaseData` is described as optional. However for the German market the content of this field is not always optional.
+In the general description, the field `ftReceiptCaseData` is described as optional. However, for the German market, the content of this field is not always optional.
 
-For some special cases we need you to transmit data within the field `ftReceiptCaseData` that is later needed for the DSFinV-K export. The following table describes when and how you have to fill them.
+For some cases, we need you to transmit data within the field `ftReceiptCaseData` that is required later for the DSFinV-K export. The following table describes when and how you have to fill them.
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/data-structures/data-structures.md:40]: bit better if the instructions are written with passive:
For some cases, it is needed to transmit data within the field ftReceiptCaseData, that is required later for the DSFinV-K export. The following table describes how and when these need to be filled.

File: doc/appendix-de-kassensichv/data-structures/data-structures.md [hunk 4]
```diff
@@ -23,55 +23,55 @@ If you need to provide customer data in your request, you can send it in via the
...
 | `ReceiptNote` | `string`      | optional                         | Additional information on the receipt header. Can be sent via `ftReceiptCaseData` in JSON format. To send, add the key value pair `ReceiptNote` e.g. `"ftReceiptCaseData":"{ ..., "ReceiptNote":"123, ich bin dabei!", ... }"` | 1.3|
 | `ReceiptName` | `string`      | Mandatory if your request mapps to the DSFinV-K BON_TYPE "AVSonstige" (see `ftReceiptCase`), otherwise optional | Can be sent via `ftReceiptCaseData` in JSON format. To send, add the key value pair `ReceiptName` e.g. `"ftReceiptCaseData":"{ ..., "ReceiptName":"Sonstige Sonderwurst", ... }"` | 1.3|
 
-If you need to provide a **reference** to another systems or another cashpoints, you can add it via the field `ftReceiptCaseData` by providing it's data as shown below:
+If you need to provide a **reference** to another system or another cashpoint, you can add it via the field `ftReceiptCaseData` by providing it's data as shown below:
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/data-structures/data-structures.md:49]: by providing its data

File: doc/appendix-de-kassensichv/data-structures/data-structures.md [hunk 5]
```diff
@@ -23,55 +23,55 @@ If you need to provide customer data in your request, you can send it in via the
...
 
 ### Charge Items Entry
 
-In the general description the field `ftChargeItemCaseData` is described as optional. However for the German market the content of this field is not always optional.
+In the general description, the field `ftChargeItemCaseData` is described as optional. However, for the German market, the content of this field is not always optional.
 
-For some special cases we need you to transmit data within the field `ftChargeItemCaseData` that is later needed for the DSFinV-K export. The following table describes when and how you have to fill them.
+For some cases, we need you to transmit data within the field `ftChargeItemCaseData` that is required later for the DSFinV-K export. The following table describes when and how you have to fill them.
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/data-structures/data-structures.md:64]: For some cases, it is needed to transmit data within the field --------, that is required later for the DSFinV-K export. The following table describes how and when these need to be filled.

File: doc/appendix-de-kassensichv/data-structures/data-structures.md [hunk 6]
```diff
@@ -82,14 +82,14 @@ Following table highlights fields of the charge item that need a special handlin
...
+| `VATAmount` | `decimal (5)` | mandatory for special cases | In some special cases of taxation (e.g. car spare part in the car repair shop), the VAT is not a percentage of the net-price (NETTO) or the gross-price. For these cases, the field  `VATAmount` is mandatory and a `ftChargeItemCase` that maps to DSFinV-K UST_SCHLUESSEL 7 should be used. The value of the field `VATRate` should be set to `0.0`. | 1.3 |
 
 
 ### Pay Items Entry
 
-In the general description the field `ftPayItemCaseData` is described as optional. However for the German market the content of this field is not always optional.
+In the general description, the field `ftPayItemCaseData` is described as optional. However, for the German market, the content of this field is not always optional.
 
-For some special cases we need you to transmit data within the field `ftPayItemCaseData` that is later needed for the DSFinV-K export. The following table describes when and how you have to fill them.
+For some cases, we need you to transmit data within the field `ftPayItemCaseData` that is required later for  the DSFinV-K export. The following table describes when and how you have to fill them.
```
Comment (DoxaPap) [inline doc/appendix-de-kassensichv/data-structures/data-structures.md:92]: For some cases, it is needed to transmit data within the field -------------, that is required later for the DSFinV-K export. The following table describes how and when these need to be filled.

File: doc/appendix-de-kassensichv/data-structures/data-structures.md (hunk 1 above)
Comment (aedler-ft) [reply doc/appendix-de-kassensichv/data-structures/data-structures.md:3]: oh yes, so true, thank you!

File: doc/appendix-de-kassensichv/data-structures/data-structures.md (hunk 2 above)
Comment (aedler-ft) [reply doc/appendix-de-kassensichv/data-structures/data-structures.md:9]: hm, i think i know what you mean

File: doc/appendix-de-kassensichv/data-structures/data-structures.md (hunk 3 above)
Comment (aedler-ft) [reply doc/appendix-de-kassensichv/data-structures/data-structures.md:40]: jepp

**Pull request merged**